db/data.py

```diff
@@ -58,6 +58,44 @@ def _has_key(dictionary, key):
         dictionary = dictionary[part]
     return True
 
+def check_highlevel_meta():
+    with db.engine.begin() as connection:
+        query = text("""
+                        SELECT COUNT(*)
+                        FROM highlevel_meta ;
+                    """)
+        result = connection.execute(query).fetchall()
+    return result
+
+def clean_highlevel_data():
```

Comment:
like the comment on `check_highlevel_meta`, think about the name of this method, a docstring, and tests.